pkg/reaper/pdbreaper/pdbreaper.go

```diff
@@ -146,6 +151,8 @@ func (ctx *ReaperContext) scan() error {
 		}
 
 		ctx.ClusterBlockingPodDisruptionBudgets[namespace] = append(ctx.ClusterBlockingPodDisruptionBudgets[namespace], pdb)
+
+		ctx.exposeMetric(pdb, EventReasonPodDisruptionBudgetDeleted, 0)
```

Comment:
This is prior to actual deletion

Reply:
If this is just initialization, can we put this higher up even prior to scanning?

Reply:
0 means no deletion.  Just to set the initial value of the metric